Update misc.R

dhkatz12 opened the pull request:
added functionality to handle destination file paths that contain spaces

dhkatz12 committed:
Update misc.R

biodavidjm commented on R/misc.R:
```diff
@@ -91,14 +91,24 @@ dl_read_gcp <- function(path,
     stop("The path to the bucket is wrong. Valid example: gs://bucket-name/file-name.csv")
   }else{
-    new_path <- file.path(tmpdir, basename(path))
+    new_path <- file.path(tmpdir_norm, basename(path))
```
This is wrong. It should be `tmpdir` (which was defined above). Correction coming in the next commit

biodavidjm replied:
CORRECTION: if the gs path is wrong before the file, it is not detected as it is. Fixing that issue instead. For example, this would not be detected:

```
gs://THISIWWRONGmotrpac-right-path/right-file.txt
```

biodavidjm committed:
Update dl_read_gcp: several improvements

- Minor refactor
- Check whether the file exists in GCP
- Improve verbosity
- Improve documentation

biodavidjm committed:
Improve dl_read_gcp: check gsutil path

- Check if the `gsutil` path is correct and report back to the user if it is not
- Default input file is tab delimited. If it is csv, the user must provide the `sep = ","` argument

biodavidjm reviewed (approved):
The function now works well on both mac and windows
